templates/alertmanager/alertmanager-svc-headless.yaml

```
@@ -17,10 +17,18 @@ spec:
   clusterIP: None
   publishNotReadyAddresses: true
   ports:
-    - port: {{ .Values.config.server.http_listen_port }}
+    - port: {{ .Values.config.server.grpc_listen_port }}
       protocol: TCP
       name: http-metrics
       targetPort: http-metrics
```

```suggestion
      protocol: TCP
      name: grpc
      targetPort: grpc
```
Forgot to edit this? Target port should be `grpc` (9095) not `http-metrics` (8080)